README.md

```diff
@@ -96,6 +96,18 @@ destroy` or turn off all services with `lando poweroff`.
      - Or run services separately as shown in [[https://github.com/pulibrary/figgy/blob/master/Procfile]]
    - Access Figgy at http://localhost:3000/
 
+### Parallel Tests
+
+If you'd like to run the test suite in parallel do the following:
+
+1. `bundle exec rake servers:start`
+1. `PARALLEL_TEST_FIRST_IS_1=true RAILS_ENV=test rake parallel:setup`
+1. `./bin/turbo_test`
+
+To run in parallel with working coverage, but worse output, do:
+
+1. `bundle exec parallel_rspec --first-is-1 --runtime-log tmp/turbo_rspec_runtime.log`
```

Comment:
Coverage is a primary benefit of running the full suite so I expect I'll always prefer this method. A `.bin` script that runs it would be nice.